test/integration/input_use_cases/6/run.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# exit on error
+set -e
+# turn on command echoing
+set -v
+# make sure that the current directory is the once where this script is
+cd ${0%/*}
+
+exec_str="../../../../music_box config.json"
```

Comment:
Should paths be hard-coded or should they be abstracted: exec_str = $exectuble + "config.json"?

Reply:
I'm not sure I follow - what would set `executable`?